blog/2020-11-11-open-communications.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Open"
+author: franck
+author_url: https://github.com/d4nte
+author_image_url: https://avatars1.githubusercontent.com/u/300805
+tags: [vision,opensource,comit]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+At COMIT, we believe in an open financial system - truly inclusive, censorship resistant and non-discriminating.
+
+We knew from day one that opensource software was the only way to achieve our vision, and we love opensource so it never was an issue.
+
+However, we realised that while we licensed our project with open [[1]](https://github.com/comit-network/comit-rs/blob/dev/LICENSE.md) and very open licenses [[2]](https://github.com/comit-network/ambrosia/blob/master/LICENSE), there were other obstacles on the way to declare us truly open:
+
+- Our contributor agreement was restrictive, it is now gone [[3]](https://github.com/comit-network/comit-rs/pull/3344);
+- Our design discussions moved from open GitHub [[4]](https://github.com/comit-network/spikes/) to closed basecamp, we are now back in the open with the comit-dev mail list [[5]](https://lists.comit.network/mailman/listinfo/comit-dev);
+- We had private chats, we are now discussing in public on our dev [[6]](https://matrix.to/#/!eDtJfYgJutkmKTvbOH:matrix.org?via=matrix.org) and project chats [7] on matrix;
```

Comment:
Did you not link to `[7]` on purpose? (because its 2 links?)

Reply:
It's related to @bonomat's comment. now fixed.